specification/azurestack/resource-manager/Microsoft.AzureStack/stable/2017-06-01/AzureStack.json

```diff
@@ -44,6 +44,86 @@
           "nextLinkName": "nextLink"
         }
       }
+    },
+    "/providers/Microsoft.AzureStack/cloudmanifestfiles": {
```

Comment:
can you camel case the resource type name? cloudManifestFiles?

Reply:
is this not caught by linter? surprised

Reply:
Fixed